README.md

```diff
@@ -65,3 +66,8 @@ your case on why your shortname should replace the existing short names.
 ## Contact
 
 - IRC: #[containers](irc://irc.freenode.net:6667/#containers) on freenode.net
+
+
+[podman-gh]:      https://github.com/containers/podman  "GitHub: containers/podman"
+[docker-cli-gh]:  https://github.com/docker/cli         "GitHub: docker/cli"
+[gnu-bash]:       https://www.gnu.org/software/bash/    "GNU Bash (www.gnu.org)"
```

Comment:
I don't think bash is a good contact.

Reply:
In my latest rebase I removed the link to the gnu.org bash site.